tests/unit/components/PageFooter.spec.js

```diff
@@ -1,17 +1,39 @@
-import { shallowMount } from '@vue/test-utils';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import BootstrapVue from 'bootstrap-vue';
 import PageFooter from '../../../components/PageFooter.vue';
 
+const localVue = createLocalVue();
+localVue.use(BootstrapVue);
+
+const dummyStore = {
+  state: {
+    footer: {
+      links: [
+        { url: 'https://www.example.org', text: 'Example link' },
+        { url: 'https://www.europeana.eu', text: 'Europeana link' }
+      ]
+    }
+  }
+};
+
 const factory = () => shallowMount(PageFooter, {
+  localVue,
   mocks: {
-    $t: () => {}
+    $t: () => {},
+    $store: dummyStore
   }
 });
 
+const sleep = (milliseconds) => {
+  return new Promise(resolve => setTimeout(resolve, milliseconds));
+};
+
 describe('components/search/PageFooter', () => {
-  it('it exists', () => {
+  it('it contains elements for each link', async () => {
     const wrapper = factory();
-
-    const footer = wrapper.find('footer');
-    footer.isVisible().should.equal(true);
+    await sleep(20);
```

Comment:
Why is this needed? If I remove it, the test still passes.

Reply:
Not needed any more, it was needed when the links were only retrieved on the client as that happened only after the component was rendered.

Have removed it.